src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/views/processor_ui_grok.html

```diff
@@ -11,6 +11,16 @@
   <pre>{{ fieldData }}</pre>
 </div>
 <div class="form-group">
-  <label>Pattern:</label>
+  <label>
+    Pattern:
+    <a
+      aria-label="Grok Pattern Help"
+      tooltip="Grok Pattern Help"
+      href="https://www.elastic.co/guide/en/logstash/current/plugins-filters-grok.html"
+      tooltip-append-to-body="true"
+      target="_blank">
+      <i aria-hidden="true" class="fa fa-question-circle"></i>
+    </a>
+  </label>
```

Comment:
Let's remove this tooltip until https://github.com/elastic/kibana/pull/6989 gets merged. I don't want to add any more hardcoded doc links that we'll just have to update later and potentially forget.